### DISCUSSION TITLE
【Apache IoTDB毕业两周年】留下你的故事，脑斧T恤送到家！

### DISCUSSION THREAD

**Discussion by AuroraLiu0823** [apache/iotdb#7324] [General]
喜大普奔，至2022年9月16日，Apache IoTDB已从Apache基金会毕业两周年啦！！【手动鼓掌】【手动放礼花】

在这一年，IoTDB实现了8次发版，交流群人数突破3000人，性能和功能的不断完善带来了更庞大也更温暖的社区小伙伴们。不管你是为IoTDB的性能与功能进步努力的开发者，还是社群中积极表达、使用IoTDB的用户，我们都向您表达对IoTDB的支持和关注的衷心感谢！

为回馈社区的大家，Apache IoTDB公众号小助手欧欧特准备**2周年小脑斧纪念T恤**准备发放(#^.^#)，在此帖留言，**建议/应用场景/达到的性能数字/与IoTDB小故事**均可，截图账户信息页及内容发送欧欧即可获得~（**欧欧微信二维码见下图**）

期待大家的故事，感谢大家见证IoTDB的成长，两周年快乐！

![origin_img_v2_bf16e1d0-03d5-4b8b-b040-4401b4e904dg](https://user-images.githubusercontent.com/111332545/190313007-ad7ac642-73ea-45a8-ac46-94e3d71db26b.gif)

**Comment by Damen957** [comment]
从其它时序数据库转战到了iotdb，这主要是因为iotdb的核心理念吸引了我，我觉得以树状结点的结构来存储时间序列的设计太酷了，而且iotdb还有自动创建元数据（开箱即用）的特点，加上高吞吐的性能，iotdb完美解决了我的需求；祝iotdb两周年快乐，未来再接再厉，继续提升并完善性能！

**Reply by BanDuWuZi** [reply]
你们是哪类公司，iotdb是什么应用场景啊

**Comment by detongz** [comment]
2022年2月给DataFun主持过一次支持分布式的开源时序数据IoTDB分享，会后给云智慧和乔老师整理了讲稿。整理讲稿的过程中发现了IoTDB分布式版本也是开源的，这个比其他时序数据库分布式版本收费要友好不少。IoTDB可作为Prometheus的外存设计也非常有趣，期待看到更多应用案例。

**Comment by Frank-KunLi** [comment]
两年前，在项目中需要用到时序数据库，评估了业界主流的时许数据库，发现IoTDB完美的满足我们的工业应用场景，后面和黄老师、金城、乔博进行了交流和探讨，收获多多。
一个小建议：在官网里面增加一些TSDB的关键字，能够让搜索引擎收录，能够使用者容易搜索到。

**Comment by MaxLearnging** [comment]
十分看好这个方向，若干年前在 APP内实践过移动设备采集+边缘计算+后端数据聚合，但端云数据如何高效协同是个基础性的问题。所幸看到IoTDB 在这个领域的开创性进展！

**Comment by pluto133** [comment]
在一次展会了解到了IoTBD. IoTDB 是物联网轻量级、高性能时间序列数据库，物联网数据收集、存储、查询与分析一体化的数据管理引擎，简单易用、低成本高性能。希望在领域内不断发展壮大。

**Comment by jack-She1l** [comment]
IoTDB写入性能高于同款数据库10倍有余，单机写入速度高达到每秒2千万。且硬盘占用是最小的，这在数据比较大的线上业务中，可能每个月会差出来1到2块硬盘。整体来看 IoTDB 无论在写入、原始数据查询还是聚合查询，都几乎是10倍的性能于竞品数据库，而且硬盘占用又小于同款数据库10倍。

**Comment by hey-sa** [comment]
在博客里偶然看到这个从清华出来的物联网数据库，后来又了解到是支持 aligned timeseries 的时序数据库，非常地酷。顺带一提，感觉公众号里排版很不错，[README](https://github.com/apache/iotdb/blob/master/README_ZH.md) 怎么没有一起做一些排版美化呢

**Comment by willsonyang0723** [comment]
还是挺好用的，选了很久时序数据库，最终选择了iotdb

**Comment by jianguo888** [comment]
十分看好这个方向。IoTDB在写入、数据库查询中的所有原始数据，几乎都是10倍的于竞品数据库查询原始数据。

**Comment by Cossack9989** [comment]
学Java代码审计时审计过IoTDB，代码质量挺好哦

**Comment by maskcao** [comment]
之前学习数据库时看到过IoTDB，去了解了时序数据库，然后有在关注和学习。

**Comment by tiammomo** [comment]
之前学习InfluxDB的时候了解到了IotDB，发现是国产的时序数据库，一直都很期待其发展。最近发现了这个活动后，仔细看了看发现进步非常迅速，以后碰到相关问题优先考虑尝试一下IotDB，希望国产时序数据库越来越好！hhhhhhh

**Comment by maskcao** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by irvine0109** [comment]
第一次接触数据库引擎，第一次写Java栈的code，第一次深度参与开源社区，以及，第一次与隔壁学校协作，2333，感谢Apache IoTDB社区，希望在未来的岁月里相互成就，感恩一同前行。

**Comment by maskcao** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
